packages/contracts/test/10_unit-testing/12_plugin-setup.ts

```diff
@@ -495,6 +495,31 @@ describe('MultisigSetup', function () {
         ],
       ]);
     });
+
+    it('returns the permissions expected for the update from build 3', async () => {
```

Comment:
to build 3 instead of from build 3 (imo)

Reply:
it is testing updating from build 3 to build 3, this is not realistic but it was to check that when upgrading from bigger versions, permissions list is empty